jerry-core/debugger/debugger.h

```diff
@@ -29,6 +39,11 @@
 #define JERRY_DEBUGGER_MESSAGE_FREQUENCY 5
 
 /**
+* Sleep time in milliseconds between each jerry_debugger_receive() call
+*/
+#define JERRY_DEBUGGER_TIMEOUT 100
```

Comment:
as this define is used in the source file only, it can be moved there, too (and don't forget the proper indentation of the comment block)